scipy/optimize/tests/test_optimize.py

```diff
@@ -1540,6 +1540,14 @@ def test_brent(self):
         x = optimize.brent(self.fun, brack=(-15, -1, 15))
         assert_allclose(x, self.solution, atol=1e-6)
 
+        with pytest.raises(ValueError, match='Bracketing values (xa, xb, xc) do not fulfill this\
```

Comment:
The match statement does not need to have the whole message. A fraction of it would work. In this case, matching for `(f(xb) < f(xa)) and (f(xb) < f(xc))` would be enough. It would also fix the line length/formatting issue.